library/src/scripts/forms/themeEditor/ColorPicker.tsx

```diff
@@ -56,15 +63,20 @@ export default function ColorPicker(props: IColorPicker) {
     const onTextChange = e => {
         const colorString = e.target.value;
         helpers.setTouched(true);
+
         if (stringIsValidColor(colorString)) {
             setValidColor(colorString); // Only set valid color if passes validation
+            errorHelpers.setValue(undefined);
```

Comment:
Aren't you missing a helpers.setValue(colorString) here?

Reply:
Fixed.